pkg/delete/deleteAll_test.go

```diff
@@ -0,0 +1,69 @@
+//go:build unit
```

Comment:
File name is camel case

Reply:
moved to https://github.com/Dynatrace/dynatrace-configuration-as-code/pull/1726